package.json

```
@@ -39,6 +39,7 @@
     "build": "webpack --mode production",
     "lint": "eslint --cache --format codeframe src",
     "test": "jest",
+    "format:json": "node scripts/format-json.js",
```

```suggestion
    "data:nais:ecogram": "node data/nais/ecogram.js",
```
Can you also add the source GeoJSON data to `data/nais/ecograms.json` or `data/nais/ecograms/1.json`?

---

Transformed data should be written and committed to `data/ecograms.json`.